src/lib/auth.ts

```diff
@@ -93,10 +92,10 @@ export const DEVICE_API_KEY_PERMISSIONS = [
 
 	'resin.service.read?application/canAccess()',
 
-	'resin.service_install.read?installs__service/canAccess()',
+	'resin.service_install.read?device/canAccess()',
```

Comment:
This granted to a device api key read access to all service_installs & device_service_environment_variables of all devices of the application.